src/clients/HydrogenClient.ts

```diff
@@ -106,6 +120,21 @@ class HydrogenClient {
       data: result.data.map(formatCrossChainTransferDetailed),
     }
   }
+
+  async getFeeQuote(req: GetFeeQuoteRequest): Promise<GetFeeQuoteResponse> {
+    this.checkState();
+    const request = this.apiManager.path('fee_quote', {}, {
+      ...req,
+      include_tx: true,
+    })
+    const response = await request.get();
+    const result = response.data;
+
+    return {
+      ...result,
+      data: result.data.map(formatFeeQuote),
```

Comment:
The json data returned from the fee_quote endpoint is not an array.
<img width="1281" alt="Screenshot 2022-08-29 at 3 13 37 PM" src="https://user-images.githubusercontent.com/65583522/187145867-e5cf8308-0c5e-4db2-b2f4-c06af2c174f2.png">

So line 133 - 136 should be changed to this instead:
```
return formatFeeQuote(result)
```